src/Pokemon.java

```diff
@@ -0,0 +1,68 @@
+import java.util.List;
+
+public class Pokemon {
```

Comment:
Deze klasse mag abstract worden, want je wilt niet een algemene Pokemon zonder type kunnen maken. In dat geval kun je ook abstracte methodes toevoegen, zoals de `getAttacks()` methode die je nu in iedere subklasse hebt staan